extension/page-detect.js

```diff
@@ -1,7 +1,7 @@
 /* globals utils */
 
 window.pageDetect = (() => {
-	const isGist = () => location.hostname === 'gist.github.com';
+	const isGist = () => location.hostname.startsWith('gist.') || location.pathname.startsWith('gist/');
```

Comment:
Fair enough. I've checked it doesn't exist but I didn't go so far to register it just to prove a point! 😆 It's good that you've checked it! 👍